YouVsVirus/Assets/Scenes/Levelgethome/EndLevelControllerLevelgethome.cs

```diff
@@ -9,6 +10,12 @@
 /// </summary>
 public class EndLevelControllerLevelgethome : EndLevelControllerBase
 {
+    public TMP_Text failText = null; //  or make public and drag
+    public override void EndLevel()
+    {   
+        CanvasFail.SetActive(true);
+        failText.text = "You pressed the exit key.";
+    }
```

Comment:
> 
> 
> > I tried with pausing the game, but then if I press "Retry" the game is still paused. Any ideas how to fix that?
> 
> The retry button should then call Unpause when pressed.

Doing this in Retry() is easy, since this function is in the base class. We also have to do it for Continue() in every level...

Reply:
Done